extensions/amp-story/1.0/amp-story.js

```diff
@@ -1642,8 +1634,36 @@ export class AmpStory extends AMP.BaseElement {
   onBookendStateUpdate_(isActive) {
     this.toggleElementsOnBookend_(/* display */ isActive);
     this.element.classList.toggle('i-amphtml-story-bookend-active', isActive);
+    this.setHistoryState_(HistoryStates.BOOKEND_ACTIVE, isActive);
+  }
+
+  /**
+   * Updates the value for a given state in the window history.
+   * @param {string} stateName
+   * @param {string|boolean} value
+   */
+  setHistoryState_(stateName, value) {
+    const {history} = this.win;
+    const state = getState(history) || {};
+    const newHistory = Object.assign({}, /** @type {!Object} */ (state),
+        {[stateName]: value});
+
+    history.replaceState(newHistory, '');
```

Comment:
Is that second parameter needed? It looks like it's ignored by browsers for now, but if we can remove it or set it to undefined, maybe it's safer? If the linter requires it, then fine : )

Reply:
It is needed by the compiler :| it wouldn't let me set it to undefined either.